content/doc/book/pipeline/jenkinsfile.adoc

```diff
@@ -420,6 +420,45 @@ apply to all steps within the Pipeline.
 given environment variables to steps within the `stage`.
 
 
+==== Setting environment variables dynamically 
+
+In the case where environment variable need to be set dynamically at run time this can be done with the use of a shell scripts (sh), Windows Batch Script (bat) or PowerShell Script (powershell). Each script can either returnStatus or returnStdout.  link:https://jenkins.io/doc/pipeline/steps/workflow-durable-task-step[More information on scripts].
```

Comment:
I'd like it if the steps (`sh`, `bat`, `powershell`) and the parameter names (`returnStatus`, `returnStdout`) were backtick-quoted.